spec/rubocop/cop/style/method_called_on_do_end_block_spec.rb

```diff
@@ -5,62 +5,57 @@
 
   context 'with a multi-line do..end block' do
     it 'registers an offense for a chained call' do
-      inspect_source(cop, <<-END.strip_indent)
+      expect_offense(<<-RUBY.strip_indent)
         a do
           b
         end.c
-      END
-      expect(cop.offenses.size).to eq(1)
-      expect(cop.highlights).to eq(['end.c'])
+        ^^^^^ Avoid chaining a method call on a do...end block.
```

Comment:
Are there cases where a multi-line highlight is ever actually displayed in the output? I've never seen it myself. As far as I could tell,  a multi-line highlight ends up just highlighting the first line